dipy/reconst/ivim.py

```diff
@@ -286,7 +286,7 @@ def __init__(self, gtab, split_b_D=400.0, split_b_S0=200., bounds=None,
             e_s += "bounds. Please update to Scipy 0.17 to use bounds"
             raise ValueError(e_s)
         elif self.bounds is None:
-            self.bounds = ((0., 0., 0., 0.), (np.inf, .3, 1., 1.))
+            self.bounds = ((0., 0., 0., 0.), (np.inf, .3, 0.2, 1.))
```

Comment:
Could you please define these bounds as module-level global variables, so that we can reuse them in https://github.com/nipy/dipy/blob/master/dipy/reconst/ivim.py#L661 and other places? Also, we need to change the documentation (e.g., https://github.com/nipy/dipy/blob/master/dipy/reconst/ivim.py#L216) accordingly.